service_user.go

```diff
@@ -78,6 +85,24 @@ func (h *ServiceUsersHandler) Get(project, serviceName, username string) (*Servi
 	return nil, err
 }
 
+// Put modifies the given Service User in Aiven.
+func (h *ServiceUsersHandler) Put(project, service, username string, password string) (*ServiceUser, error) {
```

Comment:
Can you please change a method name from `Put` to `Update` for consistency with other handlers?